config/default/assisted-service-patch-volumes.yaml

```diff
@@ -10,7 +10,8 @@ spec:
         app: assisted-service
     spec:
```

Comment:
After testing locally, I think the spec should be:

```yaml
    spec:
      containers:
      - name: assisted-service
        volumeMounts:
        - mountPath: /data
          name: bucket-filesystem
      volumes:
      - name: bucket-filesystem
        persistentVolumeClaim:
          claimName: bucket-pv-claim
```

bucket-pv-claim matches the guidance we give here: https://github.com/openshift/assisted-service/blob/master/config/manifests/bases/assisted-service-operator.clusterserviceversion.yaml#L44